gcloud_dataproc/v02/create_cluster_without_VEP.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+
+from gcloud_dataproc.utils.machine_types import MACHINE_TYPES, get_cost
+
+def create_cluster(cluster='without-vep', machine_type='n1-highmem-8', max_idle='30m', num_workers=2, num_preemptible_workers=0, region=None):
+    cost1 = get_cost(machine_type=machine_type, hours=1, is_preemptible=False) * num_workers
+    cost2 = get_cost(machine_type=machine_type, hours=1, is_preemptible=True) * num_preemptible_workers
+    print("Cost: $%0.2f/h + $%0.2f preemptible/h = $%s / hour" % (cost1, cost2, cost1 + cost2))
+
+    # create cluster
+    command = f"""gcloud beta dataproc clusters create {cluster} \
+        {f'--region={region}' if region else ''} \
+        --max-idle={max_idle}s \
```

Comment:
```suggestion
        --max-idle={max_idle} \
```
Tiny thing but I dont think the s is needed here.